tasks/system/Linux.yml

```diff
@@ -1,117 +1,126 @@
 ---
 # tasks file for sonarqube pre install
 
-- name: "Become section"
+- name: Become section
   block:
 
-  - name: "Stop SonarQube if running"
-    service:
-      name: "sonarqube"
-      state: "stopped"
-    ignore_errors: True
+  - name: Install epel
+    package:
+      name: '{{ epel_packet }}'
```

Comment:
Will it work on Debian family?

Reply:
No. By readme right now:
 - **Supported OS**:
   - CentOS
     - 7
   - RHEL
     - 7